src/main/java/com/gregtechceu/gtceu/api/block/MaterialBlock.java

```diff
@@ -279,30 +275,4 @@ public boolean replaceWithFramedPipe(Level level, BlockPos pos, BlockState state
         }
         return false;
     }
-
-    @Override
-    public void entityInside(BlockState state, Level level, BlockPos pos, Entity entity) {
-        if (this.tagPrefix == TagPrefix.frameGt && entity instanceof LivingEntity livingEntity) {
-            double currentAccel = 0.15D * (livingEntity.getDeltaMovement().y < 0.3D ? 2.5D : 1.0D);
-            double currentSpeedVertical = 0.9D * (livingEntity.isInWater() ? 0.4D : 1.0D);
-            Vec3 deltaMovement = livingEntity.getDeltaMovement();
-            livingEntity.resetFallDistance();
-            float f = 0.15F;
-            double d0 = Mth.clamp(deltaMovement.x, -f, f);
-            double d1 = Mth.clamp(deltaMovement.z, -f, f);
-            double d2 = Math.max(deltaMovement.y, -f);
-            if (d2 < 0.0 && !livingEntity.getFeetBlockState().isScaffolding(livingEntity) &&
-                    livingEntity.isSuppressingSlidingDownLadder() &&
-                    livingEntity instanceof Player) {
-                d2 = Math.min(deltaMovement.y + currentAccel, 0.0D);
-            }
-            if (livingEntity.horizontalCollision) {
-                d2 = 0.3;
-            }
-
-            deltaMovement = new Vec3(d0, d2, d1);
-
-            entity.setDeltaMovement(deltaMovement);
-        }
-    }
```

Comment:
why remove?

Reply:
the entire logic just doesnt work atm, we need to figure out a better method for making them exactly ladder like